### PR TITLE
Revert shell/platform/common/BUILD.gn

### PR DESCRIPTION
**Pull request by bwikbs**
For running embedder's unit test, I need x64 build.
(I am modifying the embedder API for the ``FlutterEngineGroup`` API, and I want to add a unit test.)
Unfortunately, modification of BUILD.gn are preventing that task.

This change came from [here](https://github.com/flutter-tizen/engine/commit/bb85756e045cac6aa8c3cdcafddb9dcf44fc8083), I couldn't find a reason. [previous_work](https://github.sec.samsung.net/f-project/engine/blob/flutter-1.22-candidate.12-tizen/shell/platform/common/cpp/BUILD.gn)

Does anyone know the history?
Can I reverse this?

### CODE REVIEW

**Comment by swift-kim** [comment]
The difference is quite obvious. Compare the sizes of the embedders (libflutter_tizen*.so) and you will see the difference.

**Comment by bwikbs** [comment]
@swift-kim Oh, In that case, How about now?

**Review by swift-kim — APPROVED** [review]
@bwikbs Looks fine!

File: shell/platform/common/BUILD.gn [hunk 1]
```diff
@@ -111,6 +112,9 @@ source_set("common_cpp") {
     ":common_cpp_library_headers",
     "//flutter/shell/platform/common/client_wrapper:client_wrapper",
   ]
+  if (!build_tizen_shell) {
+    deps += [ "//flutter/shell/platform/embedder:embedder_as_internal_library" ]
+  }
```
Comment (bbrto21) [inline shell/platform/common/BUILD.gn:117]:
```suggestion

  if (!build_tizen_shell) {
    deps += [ "//flutter/shell/platform/embedder:embedder_as_internal_library" ]
  }
```

**Review by pkosko — APPROVED** [review]
LGTM

**Pull request merged**